lib/bojangles.rb

```diff
@@ -107,4 +109,8 @@ def update_emailed_status!(to:)
       file.puts to.to_json
     end
   end
+
+  def update_log_file!(to:)
+    File.open "#{todays_date}.json", 'w' do |file|
```

Comment:
Let's store the current log file in the `log` directory as well. It's easy to distinguish it since it's JSON and not zip (and of course has today's date).